immersive-web-wg-charter.html

```diff
@@ -399,15 +392,15 @@ <h3 id="w3c-coordination">
               The Web Application Security Working Group develops the
               Permissions API as well as guidance on the definition of powerful
               features, both of which might apply to the features provided by
-              the WebVR API.
+              the WebXR Device API.
             </dd>
             <dt>
               <a href="https://www.w3.org/WebPlatform/WG/">Web Platform Working
               Group</a>
             </dt>
             <dd>
               The Web Platform Working Group develops the Gamepad API that the
-              WebVR API re-uses in the context of VR controllers.
+              WebXR Device API re-uses in the context of XR controllers.
```

Comment:
No longer accurate. We may eventually loop back around to a dependency on the Gamepad API but that's not yet decided.